src/checkTargetMatchToPR.js

```diff
@@ -0,0 +1,26 @@
+'use strict'
+const semverDiff = require('semver/functions/diff')
+
+const { targetOptions } = require('./getTargetInput')
+
+const checkTargetMatchToPR = (prTitle, target) => {
+  if (target === targetOptions.major) {
+    return true
+  }
+  const expression = new RegExp(/from ([^\s]+) to (.+)/g)
```

Comment:
remove constructor, global specifier is not needed. also move outside this function as we don't need to recreate it each time.

```suggestion
  const expression = /from ([^\s]+) to (.+)$/
```